tests/functional/oauth_iframe.js

```diff
@@ -94,7 +95,8 @@ define([
       var self = this;
 
       email = TestHelpers.createEmail();
-      user = TestHelpers.emailToUser(email);
+      // removed variable user
```

Comment:
I'm not sure we need these 3 comments.
It makes some sense in the context of the diff, but when looking at this code in a week/month, I don't know if the comment will make any sense.